chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urluptime/FixDatadogUrlUptimeAssertionsJob.java

```diff
@@ -0,0 +1,146 @@
+package gov.healthit.chpl.scheduler.job.urluptime;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+import com.datadog.api.client.ApiException;
+import com.datadog.api.client.v1.model.SyntheticsAPITest;
+import com.datadog.api.client.v1.model.SyntheticsAPITestConfig;
+import com.datadog.api.client.v1.model.SyntheticsAPITestType;
+import com.datadog.api.client.v1.model.SyntheticsAssertion;
+import com.datadog.api.client.v1.model.SyntheticsAssertionOperator;
+import com.datadog.api.client.v1.model.SyntheticsAssertionTarget;
+import com.datadog.api.client.v1.model.SyntheticsAssertionType;
+import com.datadog.api.client.v1.model.SyntheticsTestOptions;
+import com.datadog.api.client.v1.model.SyntheticsTestOptionsHTTPVersion;
+import com.datadog.api.client.v1.model.SyntheticsTestOptionsScheduling;
+import com.datadog.api.client.v1.model.SyntheticsTestOptionsSchedulingTimeframe;
+import com.datadog.api.client.v1.model.SyntheticsTestRequest;
+
+import gov.healthit.chpl.scheduler.job.QuartzJob;
+import gov.healthit.chpl.util.DateUtil;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
```

Comment:
So, this will only output to the chplservice or maybe tomcat log? Normally we specify a particular log file for these jobs which makes the output easy to find. Do we care?